src/network.rs

```diff
@@ -117,119 +131,83 @@ pub struct SwapMetadata {
     taker_peer_id: PeerId,
 }
 
-impl SwapMetadata {
-    pub fn swap_id(&self) -> SwapId {
-        self.swap_id
-    }
-}
-
-#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
-pub struct SwapId {
-    inner: crate::SwapId,
-    shared: SharedSwapId,
-}
-
-impl SwapId {
-    pub fn new(shared: SharedSwapId) -> Self {
-        Self {
-            inner: crate::SwapId::default(),
-            shared,
-        }
-    }
-}
-
-impl From<SwapId> for crate::SwapId {
-    fn from(from: SwapId) -> Self {
-        from.inner
-    }
+// todo: move this from cnd to comit and import it from comit
```

Comment:
This should not be needed in nectar (is this dead code maybe?) and it is already gone from cnd as well :)